tests/unit/parsers/test.manifestjson.js

```diff
@@ -2736,4 +2738,180 @@ describe('ManifestJSONParser', () => {
       return testHomepageUrl('http://test.org', true);
     });
   });
+
+  describe('@mdn/browser-compat-data', () => {
+    const usesArrayValueForFirefoxDesktopCompat = {
+      // The linter does not check this "root" manifest property, it only
+      // checks its sub-properties, which is why we don't have to handle the
+      // array type for Firefox Desktop in the parser.
+      keys: ['browser_specific_settings'],
```

Comment:
I was double-checking in this comment was 100% true, and how we may make it more easily understandable to our future selves and I did notice that
- we are actually checking the compat data for browser_specific_settings not just its sub properties
- if I set strict_min_version to a version ealier than 48, we print the following warning (along with many many other compat warnings, it is easy to miss and I doubt that anyone will set the strict_min_version to such a old version at this time, and there are so many warnings that even I would not really try to read and fix them one by one, personally I would just set an higher strict min version :-P):
```
KEY_FIREFOX_ANDROID_UNSUPPORTE…   Manifest key not supported by the       "strict_min_version" requires Firefox for Android 47.0a1, which was released   manifest.json
D_BY_MIN_VERSION                  specified minimum Firefox for Android   before version 48 introduced support for "browser_specific_settings".
                                  version
```
-  in theory we are missing a warning because we don't do this for `support.firefox`, but as described above, setting a strict_min_version to a value smaller than 48 would print to many warning for a developer to really want to handle them)
- the main detail that concern me a bit (not at the moment and not for specifically for browser_specific_settings compat data) is that `browser_specific_settings" compat data gap show us that the gap may not always be a "not supported range":
  - Looking more closely to the gap for browser_specific_settings between 42-48 version, I did notice that the compat data that have a note that explains that before 48 the property was mandatory and not optional, but we don't take the note into account and we will always log a warning that describes it as unsupported in that version


in this PR I would say that we may just:

- tweak this comment to more explicitly describe that at this time we don't care about the gap in the compat data for this property because the gap is between two very old versions.

- maybe make this test to fail if the last version_added in the `browser_specific_settings` changes (by just checking that the last version_added is still 48 as it is now, I doubt that we will change it again any time soon, but maybe it is better to be safe than sorry)

In a separate follow up we may want to investigate in a bit more detail which compat data do have a `note` or a `flag` property, and based on that evaluate if we are handling those cases correctly already or if we would need to special case some of them.

Reply:
> * tweak this comment to more explicitly describe that at this time we don't care about the gap in the compat data for this property because the gap is between two very old versions.

done

> * maybe make this test to fail if the last version_added in the `browser_specific_settings` changes (by just checking that the last version_added is still 48 as it is now, I doubt that we will change it again any time soon, but maybe it is better to be safe than sorry)

done